### PR TITLE
Translations of "Specify" homogenised

### PR DESCRIPTION
**Pull request by dbitouze**

### CODE REVIEW

**Comment by dbitouze** [comment]
BTW, maybe it would be nice to make a full review of the infinitive forms versus imperative forms, e.g.:

```
msgid "Specify the maximum number of items to show."
msgstr "Specifier le nombre maximum d'éléments à afficher."
```
versus:
```
msgid "Specify an image. [...]"
msgstr "Spécifiez une image. [...]"
```

File: plone/app/locales/locales/fr/LC_MESSAGES/atcontenttypes.po [hunk 1]
```diff
@@ -439,7 +439,7 @@ msgstr "Restreint les résultats de recherche des collection(s) parente(s) en ut
 #. Default: "Specify whether you want to find values lesser than, greater than, equal to, or between the chosen value(s)."
 #: ATContentTypes/criteria/simpleint.py:70
 msgid "help_int_criteria_direction"
-msgstr "Indiquez si vous cherchez des valeurs plus petites, plus grandes, égales ou entre les valeur(s) choisie(s)."
+msgstr "Specifiez si vous cherchez des valeurs plus petites, plus grandes, égales ou entre les valeur(s) choisie(s)."
```
Comment (vincentfretin) [inline plone/app/locales/locales/fr/LC_MESSAGES/atcontenttypes.po:442]: Spécifiez

File: plone/app/locales/locales/fr/LC_MESSAGES/plone.po [hunk 2]
```diff
@@ -5843,15 +5843,15 @@ msgstr "Dossier spécialisé pour les données temporaires."
...
 
 #: plone.portlet.collection/plone/portlet/collection/collection.py:71
 msgid "Specify the maximum number of items to show in the portlet. Leave this blank to show all items."
-msgstr "Spécifie le nombre maximum d'éléments à afficher dans ce portlet. Laisser ce champ vide pour afficher tous les éléments."
+msgstr "Spécifier le nombre maximum d'éléments à afficher dans ce portlet. Laisser ce champ vide pour afficher tous les éléments."
 
 #: plone.app.collection/plone/app/collection/collection.py:70
 msgid "Specify the maximum number of items to show."
-msgstr "Préciser le nombre maximum d'éléments à afficher."
+msgstr "Specifier le nombre maximum d'éléments à afficher."
```
Comment (vincentfretin) [inline plone/app/locales/locales/fr/LC_MESSAGES/plone.po:5854]: Spécifier

**Comment by vincentfretin** [comment]
I think I prefer imperative form "Spécifiez" over infinitive form "Spécifier".
@laulaz What do you think?

File: plone/app/locales/locales/fr/LC_MESSAGES/atcontenttypes.po (hunk 1 above)
Comment (dbitouze) [reply plone/app/locales/locales/fr/LC_MESSAGES/atcontenttypes.po:442]: Sigh... Could I let you proceed?

**Comment by dbitouze** [comment]
> I think I prefer imperative form "Spécifiez" over infinitive form "Spécifier".

I have not a strong opinion about this: I let you decide :)

**Comment by laulaz** [comment]
👍

**Pull request closed without merging**